Formula/janet.rb

```diff
@@ -0,0 +1,17 @@
+class Janet < Formula
+  desc "Dynamic language and bytecode vm"
+  homepage "https://janet-lang.org"
+  url "https://github.com/janet-lang/janet/archive/v0.6.0.tar.gz"
+  sha256 "1226240f8ed7f5faafbff6d93e5802c7959c4b40f9212ac6f020d67ef3aa599e"
+  head "https://github.com/janet-lang/janet.git"
+
+  def install
+    ENV["PREFIX"] = prefix
+    system "make"
```

Comment:
`JANET_BUILD` needs to be defined here, to avoid countless error messages like:
```
fatal: not a git repository (or any of the parent directories): .git
```

I suggest:
```
  def install
    args = %W[
      PREFIX=#{prefix}
      JANET_BUILD=\\"ba09564\\"
    ]
    system "make", *args
    system "make", "install", *args
  end
```